### ISSUE TITLE
Default tables are inaccessible due to missing header (ATAG B.2.2.1 Level A)

### ISSUE THREAD

**Issue by andypillip** [froala/wysiwyg-editor#4324]
Froala is a great web content authoring tool. To count as accessible, it needs to satisfy criteria of the [Authoring Tool Accessibility Guidelines (ATAG)](https://www.w3.org/TR/ATAG20/), including part B, the production of accessible content.

By inserting tables without a table header, Froala is producing inaccessible tables by default, and authors need to spend extra time to render them accessible by adding a header.

[Guideline B.2.2: Guide authors to produce accessible content.](https://www.w3.org/TR/ATAG20/#gl_b22)
>  B.2.2.1 Accessible Option Prominence (WCAG):
> 
> If authors are provided with a choice of authoring actions for achieving the same authoring outcome (e.g. styling text), then options that will result in accessible web content (WCAG) are at least as prominent as options that will not. (Level A to meet WCAG 2.0 Level A success criteria; Level AA to meet WCAG 2.0 Level A and AA success criteria; Level AAA to meet all WCAG 2.0 success criteria) 

The default option can be interpreted as the prominent choice in this case.

The failed WCAG criterium is [1.3.1 Info and Relationships (Level A)](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation-programmatic)

While there might be tables that don't need headers, the overwhelming majority of tables do need them.

##### Expected behavior.
When adding a table, it has a table header by default.

##### Actual behavior.
Tables are inserted without header, authors need to add it manually.

##### Steps to reproduce the problem.
Use the [Froala Demo](https://froala.com/wysiwyg-editor/) and insert a table.

##### Editor version.
4.0.4

##### OS.
Fedora Linux 34

##### Browser.
Firefox 91

**Comment by AkshayCM** [comment]
Thank you for your feedback. The request has been reported to the product management team for evaluation and consideration for an upcoming release.

**Comment by zahidkarim24** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
